examples/dt/uni04delta-ipv6/control-plane.md

```diff
@@ -3,6 +3,40 @@
 ## Assumptions
 
 - A storage class called `local-storage` should already exist.
+- Cluster observability operator is already deployed. If not, follow the
+  steps found [below](#cluster-observability-operator).
+
+### Cluster observability operator
+
+Cluster Observability Operator must be installed as it is required by OpenStack
+Telemetry operator. If not installed, the below steps can be followed
+
+```bash
+cat > subscription.yaml << EOF
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: observability-operator
+  namespace: openshift-operators
+  labels:
+    operators.coreos.com/observability-operator.openshift-operators: ""
+spec:
+  channel: development
+  installPlanApproval: Automatic
+  name: cluster-observability-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+
+# Apply the cr
+oc apply -f subscription.yaml
+
+# Wait for the deployment to be ready
+oc wait deployments/observability-operator --for condition=Available \
+    --timeout=300s
+```
```

Comment:
In theory this could be handled in this DT's Kustomize templates and could be added as a stage in the associated https://github.com/openstack-k8s-operators/architecture/blob/main/automation/vars/uni04delta-ipv6.yaml

Reply:
Is it automated elsewhere when you tested it downstream? 

I have no objection to this PR otherwise.

Reply:
++ I would like to see that we try to consolidate on the automation here. 

For context, currently observability-operator is installed by `deploy-edpm.yml` which does `ansible.builtin.import_playbook: playbooks/02-infra.yml` - the `02-infra.yml` includes ci-framework role `openshift_obs` if `cifmw_deploy_obs: true`. AFICT only uni01alpha enabled the `cifmw_deploy_obs` option.

For this change to work, we would have to update the uni04delta-ipv6 job, toggling the same option.

I would rather see we implement it as suggested by Andrew here.

Reply:
It is automated in the downstream job, but I don't mind moving it here.

Reply:
I ended up including a yaml file containing the COO subscription and I added that into the control-plane's kustomization.yaml. I hope that works for you. I'll remove the line which deploys COO from the downstream MR and I'll do a recheck on the downstream testproject MR (both MRs depend on this PR) to test everything still works. I'll report back when I see it pass the CI.